### PR TITLE
Fixed invalid crash report

### PR DESCRIPTION
**Pull request by danishjafri88**
## Pull Request Checklist

- [ ] My patch has gone through review and I have addressed review comments
- [ ] My patch has a standard commit message that looks like `Fix #123: This fixes the shattered coffee cup!`

## Screenshots

If your patch includes user interface changes that you would like to suggest or that you would like UX to look at, please include them here.

## Notes for testing this patch

If useful, please leave notes for QA, explaining what this patch changes and how it can be best tested and verified.

### CODE REVIEW

File: Client.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3389,6 +3389,7 @@
 					};
 					27F443942135E11200296C58 = {
 						CreatedOnToolsVersion = 10.0;
+						DevelopmentTeam = CZ8C3324BS;
```
Comment (jhreis) [inline Client.xcodeproj/project.pbxproj]: Remove team id

**Review by jhreis — CHANGES_REQUESTED** [review]

File: Client.xcodeproj/project.pbxproj (hunk 1 above)
Comment (danishjafri88) [reply Client.xcodeproj/project.pbxproj]: Done

File: Client.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -4860,6 +4860,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				COPY_PHASE_STRIP = NO;
 				DEBUG_INFORMATION_FORMAT = dwarf;
+				DEVELOPMENT_TEAM = "";
```
Comment (jhreis) [inline Client.xcodeproj/project.pbxproj]: Can you delete these actual lines? This still overrides the xcconfig files, if it is empty 😄

Comment (danishjafri88) [reply Client.xcodeproj/project.pbxproj]: Oops ! removed

**Review by jhreis — APPROVED** [review]

**Pull request merged**